src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working ;( ");

```

Comment:
The message in the 'stopWork' method should be more informative and indicate that it is an 'Excavator' that has stopped working. Consider specifying the machine type in a consistent format, for example: 'Excavator has stopped working.'